README.md

```diff
@@ -1,3 +1,16 @@
+<p align="right">
+<img src="https://img.shields.io/maven-central/v/com.palantir.witchcraft.api/witchcraft-logging-api-objects" alt="Maven Central">
+<a href="https://opensource.org/licenses/Apache-2.0"><img src="https://img.shields.io/badge/License-Apache%202.0-lightgrey.svg" alt="License"></a>
+<a href="https://autorelease.general.dmz.palantir.tech/palantir/witchcraft-api"><img src="https://img.shields.io/badge/Perform%20an-Autorelease-success.svg" alt="Autorelease"></a>
+</p>
+
 # witchcraft-api
 
 This repository contains the Conjure definitions for the witchcraft health and logging APIs.
+
+# Intellij Plugin
+![Version](https://img.shields.io/jetbrains/plugin/v/17344) ![Downloads](http://phpstorm.espend.de/badge/17344/downloads) ![Rating](https://img.shields.io/jetbrains/plugin/r/stars/17344)
```

Comment:
Can we make these buttons link to the plugin, i.e. https://plugins.jetbrains.com/plugin/17344-witchcraft-logging?

Reply:
Great idea :-)